pyjs9/__init__.py

```diff
@@ -225,19 +231,21 @@ def send(self, obj, msg='msg'):
         >>> js9.send({'cmd': 'SetColormap', 'args': ['red']})
         'OK'
         """
-        if obj == None:
+        if obj is None:
             obj = {}
         obj['id'] = self.id
         jstr = json.dumps(obj)
         try:
-            url = urllib.urlopen(self.host + '/' + msg, jstr)
+            url = urlopen(self.host + '/' + msg, jstr.encode('utf8'))
         except IOError as e:
-            raise IOError,  "{0}: {1}".format(self.host, e.strerror)
+            raise IOError("{0}: {1}".format(self.host, e.strerror))
+
         urtn = url.read()
+
         if urtn[0:6] == 'ERROR:':
-            raise ValueError, urtn
+            raise ValueError(urtn)
         try:
-            res = json.loads(urtn, object_hook=_decode_dict)
+            res = json.loads(urtn.decode('utf8'), object_hook=_decode_dict)
```

Comment:
I'm not sure the UTF8 encode and decode are correct here ... this was just a try to make it work on Python 3.
The problem I have is that I don't have working examples of responses I can check .... if I had that for the `master` branch I could probably make it work for Python 2 and 3 in this branch.